upgrade/helpers/tasks.py

```diff
@@ -337,6 +339,17 @@ def post_upgrade_test_tasks(sat_host, cap_host=None):
     # Execute capsule sync task , after the upgrade is completed
     if cap_host:
         execute(capsule_sync, cap_host, host=sat_host)
+    # Execute task for template changes required for discovery feature
+        execute(
+            setup_foreman_discovery,
+            sat_version=sat_version,
+            host=sat_host
+        )
+    # Removing the original manifest from Default Organization (Org-id 1),
+    # to allow test-cases to utilize the same manifest.
+    logger.info("Removing the Original Manifest from Default Organization")
+    execute(hammer, 'subscription delete-manifest --organization-id 1',
```

Comment:
I think this may affect existence and scenario tests. Because we are not adding manifest in any scenario tests.
we should upload another lightweight manifest in image itself instead of deleting it from here.

Otherwise this should be a temporary fix

Reply:
No , I say we handle the uploading of manifest in the respective scenario tests itself, as more and more tests come , we may need to run those in separate organizations and then having a original manifest in default org will be problematic.
For Existence would there be a way to skip this, as we skip for updated location and organization.

Reply:
Ok, But we should remember and let folks know that whenever they need contents, upload manifest.